### PR TITLE
[#6039] Added support for Rocky Linux 8 (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1610,9 +1615,7 @@ if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos")
   else()
     message(FATAL "Unsupported CentOS major version: ${IRODS_LINUX_DISTRIBUTION_VERSION_MAJOR}")
   endif()
-elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux")
-  set(CPACK_RPM_${IRODS_PACKAGE_COMPONENT_ORACLE_NAME}_PACKAGE_REQUIRES "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-server = ${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}, unixODBC")
-elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse")
+elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux" OR IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "rocky" OR IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse")
```
Comment (trel) [inline CMakeLists.txt:1618]: suggestion...

`elseif (IRODS_LINUX_DISTRIBUTION_NAME IN_LIST "almalinux" "rocky" "opensuse")`

Comment (trel) [reply CMakeLists.txt:1622]: looks like that might not be legal... dang.

https://cmake.org/cmake/help/latest/command/if.html#existence-checks

Comment (korydraughn) [reply CMakeLists.txt:1622]: right.

**Comment by korydraughn** [comment]
I've verified that these changes are success in getting a running iRODS server on a Rocky Linux container.

**Comment by trel** [comment]
awesome, please # when ready.

**Comment by korydraughn** [comment]
added #

**Pull request merged**